force-app/main/default/lwc/recentSessions/recentSessions.js

```diff
@@ -0,0 +1,101 @@
+import { LightningElement, wire, track } from "lwc";
+import getServiceSessions from "@salesforce/apex/RecentServiceSessionController.getServiceSessions";
+import { loadStyle } from "lightning/platformResourceLoader";
+import { getObjectInfo } from "lightning/uiObjectInfoApi";
+import { prefixNamespace } from "c/util";
+
+import RECENT_SESSIONS_LABEL from "@salesforce/label/c.RecentSessions";
+import SUCCESS_LABEL from "@salesforce/label/c.Success";
+import USER_LABEL from "@salesforce/label/c.User";
+
+import SERVICESESSION_OBJECT from "@salesforce/schema/ServiceSession__c";
+import NAME_FIELD from "@salesforce/schema/ServiceSession__c.Name";
+import STATUS_FIELD from "@salesforce/schema/ServiceSession__c.Status__c";
+import PRIMARYSERVICEPROVIDER_FIELD from "@salesforce/schema/ServiceSession__c.PrimaryServiceProvider__c";
+
+import pmmFolder from "@salesforce/resourceUrl/pmm";
+
+const THIS_WEEK = "THIS_WEEK";
+const COMPLETE = "Complete";
+
+export default class RecentSessions extends LightningElement {
+    @track isAccordionSectionOpen = false;
+    @track sessionsData = [];
+    @track listViewNames = [];
+    serviceSessionObject = SERVICESESSION_OBJECT;
+    objectLabel;
+    objectLabelPlural;
+
+    labels = {
+        recentSessions: RECENT_SESSIONS_LABEL,
+        sucess: SUCCESS_LABEL,
+        user: USER_LABEL,
+    };
+
+    fields = {
+        name: NAME_FIELD,
```

Comment:
I think it's easier if we grab the apiName here so we do not need to when we refer to these fields later. Example... name: NAME_FIELD.fieldApiName. Also, I do not believe we are using the Name field anywhere.

Reply:
I removed the name field since we are not using it. Thanks for catching it